### ISSUE TITLE
[BITV] 9.1.4.4/4.2 - At 200% zoom, it is no longer possible to edit an image in the image editor because the "edit" button is no longer available. (1)

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#41853]
**Blocked on https://github.com/nextcloud/server/issues/42220**

Edit button should not exist at all if Image Editor is not activated on the instance.

![image](https://github.com/nextcloud/server/assets/6078378/b008dfdd-55ab-455c-bb8a-f0340db88007)

https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-c41a0aed27-v4-n2

**Comment by JuliaKirschenheuter** [comment]
@artonge could you please help us with this ticket?

**Comment by artonge** [comment]
At 200% zoom level, the device is currently considered as a mobile. In other word, the device is considered as not suited for edition. Therefore, we indeed remove the edit button from the interface.

Shall we really fix that?

**Comment by JuliaKirschenheuter** [comment]
done by https://github.com/nextcloud/viewer/pull/2102
Image Editor is not accessible and has a config switch
